### PR TITLE
Add preact-cli

### PR DESCRIPTION
**Pull request by developit**
- [x] I have read [Contribution Guidelines](https://github.com/reyronald/awesome-toolkits/blob/master/CONTRIBUTING.md) and followed everything.
- [x] I have an appropriate description with correct grammar.

preact-cli: https://github.com/developit/preact-cli

Preact CLI was in Kent's original tweet, and is a generally considered a good example of 0C bundling.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ A "toolkit" is a set of tools in any form that allows you to create applications
 * [Create React App](https://github.com/facebookincubator/create-react-app) - Create React apps with no build configuration.
 * [Create React App Typescript](https://github.com/wmonk/create-react-app-typescript) - Create React apps using typescript with no build configuration.
 * [vue-cli](https://github.com/vuejs/vue-cli) - A simple CLI for scaffolding Vue.js projects.
+* [preact-cli](https://github.com/developit/preact) - High-performance Preact PWA's in seconds.
```
Comment (reyronald) [inline README.md]: I think the link should be https://github.com/developit/preact-cli, shouldn't it?

**Comment by reyronald** [comment]
Hey, congratulations on your first PR! And thank you for contributing. 🎉 

I reviewed your changes and left a comment, check it out!

File: README.md (hunk 1 above)
Comment (reyronald) [reply README.md]: Also, the description should be "Your next Preact PWA starts in 30 seconds.", if I'm not mistaken.

Comment (developit) [reply README.md]: Yup, my bad!

**Comment by reyronald** [comment]
@developit Can you make the change so I can merge this?

**Comment by developit** [comment]
Whoops - fixed! Thanks @reyronald

**Comment by reyronald** [comment]
Awesome! Thanks!

**Pull request merged**